packages/core/src/utils/setup-package-json.ts

```diff
@@ -44,19 +44,34 @@ const arrayDiff = <T>(array: T[], comparison: T[] | undefined) => {
   return missingItems;
 };
 
+const exportsDiff = (
+  expected: NonNullable<PackageJson['exports']>,
+  actual: NonNullable<PackageJson['exports']>,
+) => {
+  const keysDiff = arrayDiff(Object.keys(expected), Object.keys(actual));
+  const valuesDiff = arrayDiff(
+    Object.values(expected).map((value) => JSON.stringify(value)),
+    Object.values(actual).map((value) => JSON.stringify(value)),
+  );
+  return keysDiff.length > 0 || valuesDiff.length > 0;
```

Comment:
Does this check if the correct value is matched up with the correct key? i.e. would it catch if two keys switched their values?

Reply:
It doesn't catch that, but it would be a silly thing to do and IMO not worth covering that case.

What would the code to handle that case look like?

Reply:
Well, like the `deepStrictEqual` that used to be there

Reply:
`diffPackageJson` doesn't know that `updatePackageJsonExports` injects extra CSS exports  after the build is finished, so the diff must be a little more forgiving than before.

Reply:
[`2cb7239` (#128)](https://github.com/seek-oss/crackle/pull/128/commits/2cb7239da18cbb9d3ebb35df6b70bacffb3fb4f6)